ogcserver/common.py

```diff
@@ -316,7 +318,7 @@ class WMSBaseServiceHandler(BaseServiceHandler):
     def GetMap(self, params):
         m = self._buildMap(params)
         im = Image(params['width'], params['height'])
-        render(m, im)
+        render(m, im, self.mapfactory.map_scale)
```

Comment:
@pthorin what happens if `map_scale` parameter is not supplied? could we have a test about this?

Reply:
It will probably fail. I thought I had a guard against this, setting the value to `1` if it isn't set, keeping the current behavior.

Reply:
```suggestion
        map_scale = self.mapfactory.map_scale if self.mapfactory is not None else 1
        render(m, im, map_scale)
```